packages/ic-http-certification/src/utils/response_header.rs

```diff
@@ -0,0 +1,95 @@
+use crate::{HttpResponse, CERTIFICATE_HEADER_NAME};
+use base64::{engine::general_purpose::STANDARD as BASE64, Engine};
+use ic_certification::HashTree;
+use serde::Serialize;
+
+/// Adds the `IC-Certificate` header to a given [`HttpResponse`]. This header is used by the HTTP Gateway
+/// to verify the authenticity of query call responses.
```

Comment:
Maybe add a link to the spec (e.g. https://internetcomputer.org/docs/current/references/http-gateway-protocol-spec/#the-certificate-header), and mention that it does not perform any validity/sanity/consistency checks on the arguments?(in particular `data_certificate` and `expr_path` can have arbitrary garbage, but the function will not fail)

Reply:
Good suggestion, thanks.